package.json

```diff
@@ -11,8 +12,16 @@
     "gulp-livereload": "~2.1.0",
     "gulp-plumber": "~0.6.3",
     "gulp-serve": "~0.2.0",
+    "insert-css": "^0.2.0",
     "jquery": "^2.1.3",
     "rimraf": "~2.2.8",
     "vinyl-source-stream": "^1.0.0"
+  },
+  "main": "src/index.coffee",
+  "browserify": {
+    "transform": ["coffeeify", "brfs"]
+  },
+  "scripts": {
+    "preview" : "browserify src/index.coffee -o ./preview/boxbits.js"
```

Comment:
Should this be done via gulpfile? If so, the browserify transform listing can stay there...